src/main/resources/avro/readmethods.avdl

```diff
@@ -14,8 +14,10 @@ to `ReferenceSet`s containing that same `Reference`. If no reference is
 specified, all `ReadGroup`s must be aligned to the same `ReferenceSet`.
 */
 record SearchReadsRequest {
-  /** If nonempty, restrict this query to reads within the given readgroups. */
-  array<string> readGroupIds = [];
+  /**
+  The ReadGroup to search.
+  */
+  string readGroupId;
```

Comment:
The interval could be so large that paging kicks in. In this case, it will be very difficult to make multiple individual requests and then merge them on the client side. It is fine that the server doesn't implement this feature, but we should not limit API. An appropriate implementation may not have this problem. Samtools can merge multiple BAMs in a small region and generate a merged BAM stream, but it is not exposed in htslib.

I think we should not change this part of the schema.